postgraas_server/postgres_instance_driver.py

```diff
@@ -1,21 +1,20 @@
 __author__ = 'sebastian neubauer'
 import docker
-import hashlib
 import psycopg2
 import time
+import socket
 
 
-def get_unique_id(connection_dict):
-    return hashlib.sha1(str(frozenset(connection_dict.items()))).hexdigest()
+def get_hostname():
+    return socket.getfqdn()
```

Comment:
Yes, you're right this is for another PR. Will revert this change in the next commit.